workspace/utilities/com/module-embed-item.xsl

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- VIDEOS (youtube, vimeo) -->
+<xsl:template match="url[driver = 'YouTube' or driver = 'Vimeo']" mode="module-embed-item">
+	<xsl:call-template name="com-embed-video" />
```

Comment:
Add all attributes parameter for external customization

Reply:
I think we should leave it as is in order to not clutter too much this template and let us use ext-attr only when necessary. This is more of a minimum use case rather than a exhaustive list of possibilities. What do you think?

Reply:
The problem is that declaration serves as documentation

Reply:
I was writing something than I realized that you guys were right haha. Will add them ;)

Reply:
Done!